share/shwap/row.go

```diff
@@ -140,6 +141,33 @@ func (r Row) verifyInclusion(roots *share.AxisRoots, idx int) error {
 	return nil
 }
 
+// MarshalJSON encodes row to the json encoded bytes.
+func (r Row) MarshalJSON() ([]byte, error) {
+	jsonRow := struct {
+		Shares []libshare.Share `json:"shares"`
+		Side   RowSide          `json:"side"`
```

Comment:
RowSide as 0,1,2 is not gonna be understandable for API consumers, like lumina. I wonder if we should make them readable strings instead in JSON.